model/team.go

```diff
@@ -23,6 +23,7 @@ const (
 	TEAM_EMAIL_MAX_LENGTH           = 128
 	TEAM_NAME_MAX_LENGTH            = 64
 	TEAM_NAME_MIN_LENGTH            = 2
+	TEAM_PRIVATE                    = "P"
```

Comment:
Can we move this to the top, next to "I" (team_invite) just to group them together?